app/views/organizations/index.html.erb

```diff
@@ -1,3 +1,6 @@
+<div class="alert alert-primary">
+  "Old Reports" will be removed on October 1, 2020. If there's data you'd like to see in the new “Reports” section, let us know at team@simple.org 
+</div>
```

Comment:
Can you extract this to a helper or maybe a partial somewhere, since it is repeated three times?